components/data/heroes.ts

```diff
@@ -0,0 +1,120 @@
+import { Card, Cards } from "nextra-theme-docs";
+import Image from "next/image";
+
+const heroes = [
+  {
+    name: "DHH",
+    description:
+      "Creator of Ruby on Rails, Co-owner & CTO of 37signals (Basecamp & HEY), NYT best-selling author, and Le Mans 24h class-winner.",
+    link: "https://twitter.com/dhh",
+    handle: "@dhh",
+    imageURL: "/assets/heroes/dhh.jpeg",
+  },
+  {
+    name: "Sarah Drasner",
+    description:
+      "Director of Eng, Web Infra @google , O'Reilly Author • OpenJS Board Member",
+    link: "https://twitter.com/sarah_edo",
+    handle: "@sarah_edo",
+    imageURL: "/assets/heroes/sarah.jpeg",
+  },
+  {
+    name: "Lee Robinson",
+    description:
+      "Helping developers build a faster web. Teaching about web development, serverless, and React / Next.js.",
+    link: "https://twitter.com/leeerob",
+    handle: "@leeerob",
+    imageURL: "/assets/heroes/lee.jpeg",
+  },
+  {
+    name: "Franz Kati",
+    description:
+      "Udemy instructor with 75k+ Students. Teaching about building a successful career in tech, with emphasis on growth and standing out as a software engineer",
+    link: "https://twitter.com/bahdcoder",
+    handle: "@bahdcoder",
+    imageURL: "/assets/heroes/Franz.jpeg",
+  },
+  {
+    name: "Sara Soueidan",
+    description:
+      "Inclusive UI/design engineer, int'l speaker, author, trainer. • Creator of the Practical Accessibility course http://practical-accessibility.today ",
+    link: "https://twitter.com/SaraSoueidan",
+    handle: "@SaraSoueidan",
+    imageURL: "/assets/heroes/Sara.jpeg",
+  },
+  {
+    name: "Andrei Neagoie",
+    description:
+      "run the best tech school in the world that gets people (actually) hired at top companies (1,000,000+ pros have taken a ZTM course) ",
+    link: "https://twitter.com/AndreiNeagoie",
+    handle: "@AndreiNeagoie",
+    imageURL: "/assets/heroes/Andrei.jpeg", 
+  },
+  {
+    name: "Sebastian Sigl",
+    description:
+      " Passionate software engineer @Adevinta, Building websites in public, Tweeting about backend & frontend, Free coaching, Ex @eBay with 8 years work experience",
+    link: "https://twitter.com/sesigl",
+    handle: "@sesigl",
+    imageURL: "/assets/heroes/seb.jpeg", 
+  },
+];
```

Comment:
The plan is to move this into a separate folder called 'data' so that we can import it into component/heroes